data/json/monsters/zed_misc.json

```diff
@@ -1476,6 +1533,57 @@
     ],
     "armor": { "bash": 5, "cut": 3, "bullet": 2, "electric": 2 }
   },
+  {
+    "id": "mon_zombie_ocular",
+    "type": "MONSTER",
+    "name": { "str": "ocular zombie" },
+    "description": "This otherwise-regular zombie's head is absolutely filled to the brim with eyes, half of them frantically swiveling in their orbits. The other half are staring directly at you.",
```

Comment:
```suggestion
    "description": "This otherwise-regular zombie's head is absolutely filled to the brim with eyes, half of them frantically swiveling in their orbits.  The other half are staring directly at you.",
```